metron-interface/metron-rest/src/test/resources/README.vm

```diff
@@ -128,11 +128,22 @@ The metron-rest application will be available at http://localhost:8080/swagger-u
 To run the application locally on the Quick Dev host, package the application and scp the archive to node1:
 ```
 mvn clean package
-scp ./target/metron-rest-$METRON_VERSION-archive.tar.gz root@node1:~/
+scp ./target/metron-rest-$METRON_VERSION-archive.tar.gz root@node1:$METRON_HOME
 ```
-Login to node1 and unarchive the metron-rest application.  Start the application on a different port to avoid conflicting with Ambari:
+Login to node1 and unarchive the metron-rest application:
 ```
-java -jar ./lib/metron-rest-$METRON_VERSION.jar --spring.profiles.active=vagrant,dev --server.port=8082
+ssh root@node1
+cd $METRON_HOME && tar xf ./metron-rest-$METRON_VERSION-archive.tar.gz
+```
+Start the application on a different port to avoid conflicting with Ambari:
+```
+java -jar $METRON_HOME/lib/metron-rest-$METRON_VERSION.jar --spring.profiles.active=vagrant,dev --server.port=8082
+```
+In a cluster with Kerberos enabled the start command must be run as the metron user, include references to the jaas and krb5.confg files, and kerberos support enabled:
+```
+su metron
+cd $METRON_HOME
```

Comment:
This needs to be the metron user's home.  Otherwise a permission error occurs (because root owns $METRON_HOME)